src/coll/mod.rs

```diff
@@ -568,7 +554,7 @@ impl Collection {
     // Internal insertion helper function. Returns a vec of collected ids and a possible exception.
     fn insert(&self,
               docs: Vec<bson::Document>,
-              ordered: bool,
+              options: Option<InsertManyOptions>,
```

Comment:
Here (and several other places), it made more sense to pass the options struct around directly rather than pulling out the individual fields. In this method it makes little difference, but in some others it simplifies things significantly, so I changed it here for consistency.